### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by DigitalTimK**
This PR will

- Reintroduce Code Coverage Report
- Introduce a EnvLoader-class
- Introduce a Fixure-Class and refactor the code (tests) accordingly
- Improve feedback in case of bad responses

### CODE REVIEW

File: src/Parameters/BaseParameters.php [hunk 1]
```diff
@@ -37,6 +37,8 @@ abstract public function getHTTPQuery();
      */
     protected function buildHTTPQuery($array)
     {
+        // Potential replacement by: http_build_query($queries, '', '&', \PHP_QUERY_RFC3986);
```
Comment (GhaziTriki) [inline src/Parameters/BaseParameters.php:40]: Tested for few years on a private project, RFC3986 won't work. Instead this has been ported from JS https://github.com/mconf/api-mate/blob/master/lib/vendor/bigbluebutton-api.js#L188

File: src/Util/EnvLoader.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ */
+
+namespace BigBlueButton\Util;
+
+use Dotenv\Dotenv;
+
+/**
+ * @see https://github.com/vlucas/phpdotenv
+ */
+class EnvLoader
```
Comment (GhaziTriki) [inline src/Util/EnvLoader.php:28]: Loading environment file is the responsibility of the integrator, not the library. What can you suggest better? Can we have it in the tests dir? Is there any lightweght env lib that we can use instead?

**Review by GhaziTriki — CHANGES_REQUESTED** [review]

File: src/Parameters/BaseParameters.php (hunk 1 above)
Comment (DigitalTimK) [reply src/Parameters/BaseParameters.php]: ok, remark removed.

File: src/Util/EnvLoader.php (hunk 2 above)
Comment (DigitalTimK) [reply src/Util/EnvLoader.php:28]: I am not sure what you mean? You don't like the place of the class in `src/Util` as it is just just in the tests? I can move it into the test-directory if you like to have it there. Or do you mean something else?

**Pull request merged**